pkg/network/protocols/http/ebpf_main.go

```diff
@@ -57,19 +56,17 @@ const (
 
 type ebpfProgram struct {
 	*errtelemetry.Manager
-	cfg         *config.Config
-	bytecode    bytecode.AssetReader
-	offsets     []manager.ConstantEditor
-	subprograms []subprogram
-	mapCleaner  *ddebpf.MapCleaner
+	cfg                        *config.Config
+	bytecode                   bytecode.AssetReader
+	offsets                    []manager.ConstantEditor
+	subprograms                []subprogram
+	subprogramsProbesResolvers []probeResolvers
```

Comment:
i think `probesResolvers` is a more meaningful name in this context, as they have nothing to do with the `subprograms` array, while current name suggests the opposite